bazaarvoice/emodb
# new postman tests for put uac/api-key endpoint

## What Are We Doing Here?

We have created new postman tests for put /uac/1/api-key endpoint in order to extend test coverage 

## How to Test and Verify

1. Run this PR on Jenkins with corresponding branch for CI test run update
2. Verify that tests for put uac/1/api-key have been executed and there are no test failures
3. Profit

## Risk

### Level 

`Low`

### Required Testing

`Smoke`
### Risk Summary
New unstable tests can be added into test suite of the regression test set
Some tests can create test data that might cause DB or application corruption

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
